README.md

```diff
@@ -20,9 +20,17 @@ and positive predictive value rates, and a confustion matrix.
 
 
 ## Quick Start
+Pre-work: Make sure to cd into the location of a projects folder, where you will clone this github repo.  Within the folder, cd into the WA-Testing-Tool folder. 
 1. Install code  `git clone https://github.com/cognitive-catalyst/WA-Testing-Tool.git`
 2. Install dependencies `pip3 install --upgrade -r requirements.txt`
 3. Set up parameters properly in configuration file (ex: `config.ini`). Use `config.ini.sample` to bootstrap your configuration.
+  a. In your terminal, copy the config file into a new one, `cp config.ini.sample config.ini`
+  b. Download a text editor if you don't already have one.  A good one would be atom or visual studio code.
```

Comment:
Let's not suggest an editor, just use an instruction to "Open the config file in your favorite text editor"